src/pages/RepoPage/ComponentsTab/subroute/ComponentsTable/hooks/useRepoComponentsTable.spec.tsx

```diff
@@ -128,9 +129,22 @@ describe('useRepoComponentsTable', () => {
   function setup({
     repoData = mockGetRepo,
     noData = false,
-    useParamsValue = { search: '', historicalTrend: '', components: [] },
+    useParamsValue = {
+      search: '',
+      historicalTrend: TIME_OPTION_VALUES.LAST_3_MONTHS,
+      components: [],
+    },
+  }: {
+    repoData?: any
+    noData?: boolean
+    useParamsValue?: {
+      search?: string
+      historicalTrend?: string
+      components: string[]
+    }
```

Comment:
I would pull these out into their own interface above, just to make this a bit cleaner